frame/election-provider-support/solution-type/src/lib.rs

```diff
@@ -49,6 +49,8 @@ pub(crate) fn syn_err(message: &'static str) -> syn::Error {
 ///   compact encoding.
 /// - The accuracy of the ratios. This must be one of the `PerThing` types defined in
 ///   `sp-arithmetic`.
+/// - The maximum number of voters per snapshot. This must be of type `Get<u32>`. Check <https://github.com/paritytech/substrate/issues/10866>
```

Comment:
Some more context of how this is used will be helpful.

Reply:
Not sure if what I did is good? I basically explained in the rustdoc what the GitHub Issue explained.